releases-hub-gradle-plugin/src/main/java/com/releaseshub/gradle/plugin/ReleasesHubGradlePlugin.kt

```diff
@@ -52,6 +52,7 @@ class ReleasesHubGradlePlugin : Plugin<Project> {
             upgradeDependenciesTask.gitHubRepositoryOwner = extension.gitHubRepositoryOwner
             upgradeDependenciesTask.gitHubRepositoryName = extension.gitHubRepositoryName
             upgradeDependenciesTask.gitHubWriteToken = extension.gitHubWriteToken
+            upgradeDependenciesTask.gitHubApiUrl = extension.gitHubApiHostName
```

Comment:
The prop name on the task should be `gitHubApiHostName`

Reply:
Sorry, fixed!